autogpt_platform/backend/backend/executor/manager.py

```diff
@@ -684,7 +693,6 @@ def on_done_task(node_exec_id: str, result: object):
 
             if _graph_exec := db_client.update_graph_execution_stats(
                 graph_exec_id=graph_exec.graph_exec_id,
-                status=execution_status,
```

Comment:
@Pwuts, I hope this is not expected that we need to keep updating the graph execution status on each node execution?

Reply:
The status not per se, it will just be running. But the *stats* must be updated on every node execution finish, for frontend responsiveness. This was an explicit request from @Torantulino .